tsfile/src/main/java/org/apache/iotdb/tsfile/file/metadata/enums/TSDataType.java

```diff
@@ -98,6 +99,23 @@ public short serialize() {
     return enumToByte();
   }
 
+  public int getDataTypeSize() {
+    switch (this) {
+      case BOOLEAN:
+        return 1;
+      case INT32:
+      case FLOAT:
+        // For text: return the size of reference here

```

Comment:
Reference's length is 8 bytes. Or 16 bytes exactly (Java objects head length)

Reply:
This method is for calculating the array sizes in array pool. For text data, it should return the reference size of a Binary object. And the actual size of the Binary object won't be considered here. 
So, should it return 8?

Reply:
Yes, In 64-bit machine, it should return 8

Reply:
Thanks, update soon~

Reply:
Fixed~